app/uk/gov/hmrc/softdrinksindustrylevy/connectors/DesConnector.scala

```diff
@@ -39,16 +37,16 @@ class DesConnector(
   servicesConfig: ServicesConfig,
   persistence: SdilPersistence,
   auditing: AuditConnector)(implicit clock: Clock, executionContext: ExecutionContext)
-    extends DesHelpers(servicesConfig) with OptionHttpReads {
+    extends DesHelpers(servicesConfig) {
 
   val desURL: String = servicesConfig.baseUrl("des")
   val desDirectDebitUrl: String = servicesConfig.baseUrl("des-direct-debit")
   val serviceURL: String = "soft-drinks"
   val cache: TMap[String, (Option[Subscription], LocalDateTime)] = TMap[String, (Option[Subscription], LocalDateTime)]()
 
   // DES return 503 in the event of no subscription for the UTR, we are expected to treat as 404, hence this override
-  implicit override def readOptionOf[P](implicit rds: HttpReads[P]): HttpReads[Option[P]] = new HttpReads[Option[P]] {
-    def read(method: String, url: String, response: HttpResponse): Option[P] = response.status match {
+  implicit def HttpReads[A](implicit rds: HttpReads[A]): HttpReads[Option[A]] = new HttpReads[Option[A]] {
```

Comment:
Can we write a uni test for this connector or check if we have an existing unit test to verify in case we get 503 it will be treated as 404?